pkg/providers/pricing/metrics.go

```diff
@@ -34,7 +34,7 @@ var (
 		prometheus.GaugeOpts{
 			Namespace: metrics.Namespace,
 			Subsystem: cloudProviderSubsystem,
-			Name:      "instance_price_estimate",
+			Name:      "instance_type_price_estimate",
```

Comment:
```suggestion
			Name:      "instance_type_price",
```

Thinking about this now. Do you think there's any reason to label the metric as estimate and not just put price? It shortens the name and the description mentions that this is an estimate.

Reply:
I'd like to leave `estimate` as it's more than likely not the actual price paid for that spot instance, but rather what Karpenter thinks the market price is.